packages/flutter/test/material/floating_action_button_test.dart

```diff
@@ -171,6 +171,7 @@ void main() {
   testWidgets('Floating Action Button elevation when highlighted - effect', (WidgetTester tester) async {
     await tester.pumpWidget(
       MaterialApp(
+        theme: ThemeData.light().copyWith(useMaterial3: true),
```

Comment:
I think it'd be easy to miss one of these. Consider creating a material 3 theme in `setUp` or `setUpAll`